### PR TITLE
Upgrade to backstage 1.7.0 & update dep to react-router-stable

### PR DESCRIPTION
**Pull request by ivangonzalezacuna**
Upgrading backstage version to 1.7.0.

With this upgrade we can then update the git submodule and avoid multiple versions hanging around. I also updated the devDependency to a react-router stable version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,20 +44,21 @@
   },
   "peerDependencies": {
     "react": "^16.13.1 || ^17.0.0",
-    "react-dom": "^16.13.1 || ^17.0.0"
+    "react-dom": "^16.13.1 || ^17.0.0",
```
Comment (heyLu) [inline package.json]: The upgrade helper uses `^17.0.2` now, should we switch to that as well?

**Review by heyLu — APPROVED** [review]
Looks good, apart from that question about the react-dom version.

File: package.json (hunk 1 above)
Comment (ivangonzalezacuna) [reply package.json]: Just checked the upstream backstage project. Seems the dependency is `^17.0.2`, but the defined peerDependency is `^17.0.0`. Probably a mismatch on their side. It should be fine to the `17.0.2` here, will test it first and then commit it

Comment (ivangonzalezacuna) [reply package.json]: Just updated the version. Nothing broke so far

**Review by heyLu — APPROVED** [review]

**Pull request merged**